downstream/assemblies/eda/assembly-eda-user-guide-overview.adoc

```diff
@@ -17,7 +17,6 @@ The following procedures form the user configuration:
 [NOTE]
 
 ====
-
-API documentation for {EDAcontroller} is available at \https://<eda-server-host>/api/eda/v1/docs
-
+* API documentation for {EDAcontroller} is available at \https://<eda-server-host>/api/eda/v1/docs
+* To meet high availability demands, {EDAcontroller} shares centralized link:https://redis.io/[Redis (REmote DIctionary Server)] with the {PlatformNameShort} UI. When Redis is unavailable due to a failure, you will not be able to create projects or enable rulebook activations.
```

Comment:
Similar comment re "unavailable due to failure."

Reply:
Got it. Will remove.